hack/prepare-release.sh

```diff
@@ -1,15 +1,38 @@
 #!/bin/bash -e
+script_dir=$(dirname "$(readlink -f "$0")")
+
 version_type=$1
 old_version=$(hack/version.sh)
 new_version=$(hack/bump-version.sh $version_type)
 commits=$(git log --pretty=format:"* %s" $old_version..HEAD)
+release_notes=$(mktemp)
+
+end() {
+    rm $release_notes
+}
+
+trap end EXIT SIGINT SIGTERM SIGSTOP
 
+release-notes \
+    --format go-template:$script_dir/release-notes.tmpl \
+    --release-version $new_version \
+    --required-author kubevirt-bot \
+    --github-org nmstate \
+    --github-repo kubernetes-nmstate \
+    --start-rev $old_version \
+    --end-rev master \
+    --output $release_notes
 
 cat << EOF > version/description
 $new_version
 
 TODO: Add description here
 
+EOF
+
+cat $release_notes >> version/description
+
+cat << EOF >> version/description
```

Comment:
I think we can drop the following list of Features Bugs and Docs. The automatic release note collection just requires us to label PRs properly and then we can have the same list autogenerated.